apps/docs/content/3-components/2-library/tooltip.mdx

```diff
@@ -9,22 +9,72 @@ libraries:
   - platform: 'global'
     status: 'considering'
   - platform: 'react'
-    status: 'considering'
+    status: 'beta'
+    link: ?path=/docs/application-tooltip--docs'
 ---
 
 # Tooltip
 
-## Variants
+## Default
+
+<ComponentContainer>
+  <Tooltip label="Default Tooltip" position="top">
+    <Button variant="primary">Hover me (Top)</Button>
+  </Tooltip>
+</ComponentContainer>
 
 ## Usage
 
+<Tabs id="tab1">
+  <TabList>
+    <TabItem value="tab11">HTML</TabItem>
+    <TabItem value="tab12">Macro</TabItem>
+    <TabItem value="tab13" checked>React</TabItem>
+  </TabList>
+  <TabPanel value="tab11">
+  TBD
+  </TabPanel>
+  <TabPanel value="tab12">
+  TBD
+  </TabPanel>
+  <TabPanel value="tab13">
+    ```jsx
+    import { Tooltip, Button } from '@govie-ds/react';
+
+    <Tooltip
+      label="Default Tooltip"
+      position="top"
+    >
+      <Button variant="primary">
+        Hover me (Top)
+      </Button>
+    </Tooltip>
+
+    ```
+
+  </TabPanel>
+</Tabs>
+
 ### Best Practices
 
+- Keep the tooltip content concise and relevant. Tooltips are meant for brief contextual information.
+- Avoid overloading tooltips with large amounts of text or complex HTML structures.
+- Ensure the tooltip is accessible by providing appropriate ARIA attributes (`aria-label`, `aria-describedby`).
```

Comment:
I didn't see the aria attributes on the component, not sure how the developper will use it.

Reply:
Good catch!